api/stats.js

```diff
@@ -1,24 +1,25 @@
 import { SnowflakeUtil, MessageEmbed } from "discord.js";
 import { config } from "../config.js";
 
-const removeParentheses = (word) => word.replace(/\(|\)/g, "");
+/*
+    Returns a snowflake which represents the time x days in the past
+    https://discord.com/developers/docs/reference#snowflakes
+*/
+const getSnowflakeFromDay = (days) => {
+    const d = new Date();
+    d.setHours(0, 0, 0, 0);
+    d.setDate(d.getDate() + days);
+    return SnowflakeUtil.generate(d);
+};
+
+const BOTS = ["elprimobot", "ChillBot"];
```

Comment:
nit: this could be an environment variable

Reply:
would this change for env?
Do you think this should just be another config property but not necessarily change per environment?

I'll move it to the config, not sure if there's a clear benefit so far.

Reply:
done, FYI: added a new property BOTS and it is expecting a comma-separated list of names, it defaults to "elprimobot" however, for the local environment you want to include ["elprimobot","ChillBot"]